README.md

```diff
@@ -1,5 +1,14 @@
 # Ngenic Sensor Platform for Home Assistant
-This platform adds sensors for Ngenic Tune nodes that report temperature and humidity.
+This platform adds sensors for Ngenic Tune nodes that report temperature and humidity if a Ngenic Track device is connected the current power and energy consumption for the current month, last month and today.
```

Comment:
I'd try to abstract it more, since you can add a lot of different sensors with the track, then we don't need to update the readme each time :) maybe:

> This platform adds sensors for Ngenic Tune & Track nodes that report temperature, humidity, power and energy consumption.

Reply:
Also update this in the `info.md`, but do not include the `Installation` section